.c8rc

```diff
@@ -0,0 +1,6 @@
+{
+  "all": true,
+  "include": ["client/out/**"],
```

Comment:
Which directories from lsp server should be included here?

Reply:
Yes, I think src should be good.
BTW., for client, why not "client/src" but "out"?

Reply:
The coverage counters act upon the transpiled output from what I've experimented with. When I changed it to only include client/src/** and server/src/**, c8 didnt output any coverage data in the report. Based on this is it still appropriate to exclude dist from server directory?

Reply:
Thank you for the information. I don't have much knowledge about it. I'm confused about why it can count the files in "client/out". The tests run on VS Code which will run the "client/dist/node/extension.js" file. The files in "client/out" is not used except test files. Perhaps only including "dist" directory should be enough?
Also, I guess the files in "connection/rest/api" can be excluded as they're generated.

Reply:
OK, I see. Looking at the coverage report, the "node/extension.ts" file shows 0. So it only counts the files that directly imported from the tests. So I guess both "client/out" and "client/dist" should be included, hopefully it can aggregate the coverage. And for server "server/dist" should be good.

Reply:
@scnwwu thanks, will update the includes to add these.